cmd/bedFormat/bedFormat.go

```diff
@@ -23,6 +23,13 @@ func bedFormat(s Settings) {
 	}
 
 	for v := range ch {
+		if s.Pad > 0 {
+			v.ChromStart -= s.Pad
+			v.ChromEnd += s.Pad
+			if v.ChromStart < 0 {
+				v.ChromStart = 0
+			}
```

Comment:
no check for chromEnd > len(chrom). Perhaps pad will require a chromsizes file?

Reply:
Yes, I think we probably want a check to make sure we don't go off the end.